components/offer/MakingOfferSingleItem.vue

```diff
@@ -87,11 +88,9 @@ const offerExpirationStoreItem = computed({
   set: v => offerStore.updateItem({ id: item.value.id, offerExpiration: v }),
 })
 
-const highestOfferPrice = computed(() => formatWithBlank(Number(item.value.highestOffer) || 0) || '--')
+const highestOfferPrice = computed(() => formatWithBlank(item.value.highestOffer))
```

Comment:
and you don't even need `formatWithBlank` just do

```suggestion
const {formatted: highestOfferPrice}  = useAmount(computed(() => value), decimals, chainSymbol, { withBlank: true } )
```

Reply:
I personally think having a wrapper function to handle the function call with passing parameters, options and then just pass the value of whatever price property is doing it right now to it is cleaner than doing the whole full function call 3 or 2 times. But sure I can do it like that if that's the preferred way.

Reply:
if you go with util function move it inside a separate file like `/utils/format/balance.ts` but dont use a composable inside 

or use composables directly